query/Inserter.java

```diff
@@ -222,17 +229,27 @@ private Attribute insertAttribute(AttributeType attributeType, ThingVariable var
     }
 
     private Relation insertRelation(RelationType relationType, ThingVariable var) {
+        Relation relation = relationType.create();
+        return extendRelation(relation, var);
```

Comment:
Are you sure we should call `extendRelation()` here and then call it again at the end of the main `insert()` method? This feels like a bug to me.